PacketDotNet/DrdaDDMPacket.cs

```diff
@@ -159,9 +159,9 @@ public List<DrdaDDMParameter> Parameters
         }
 
         /// <summary>
-        /// 
+        /// Constructor
```

Comment:
Please fold these and other improvement changes into the earlier commits when the files were introduced.